### PR TITLE
Add can_be_archived method in the Book class

### LINKED ISSUE #10: [1pt] Implement methods
1. add_item method in the Label class
    -   should take an instance of the Item class as an input
    -   should add the input item to the collection of items
    -   should add self as a property of the item object (by using the correct setter from the item object)

1. can_be_archived?() in the Book class
    -   should override the method from the parent class
    -   should return true if parent's method returns true OR if cover_state equals to "bad"
    -   otherwise, it should return false

### PR DESCRIPTION
**Pull request by aimalamiri**
## Override the can_be_archived? method in the book class

closes #10

### CODE REVIEW

File: classes/book.rb [hunk 1]
```diff
@@ -8,4 +8,9 @@ def initialize(publish_date, publisher, cover_state)
     @publiser = publisher
     @cover_state = cover_state
   end
+
+  def can_be_archived?
+    true if can_be_archived? || @cover_state == 'bad'
```
Comment (Yazino12) [inline classes/book.rb]: In this method you are trying to _`return true if parent's method returns true OR if cover_state equals to "bad"`_

- And the way you call parents method is by using the `super` method
- E.g: `super || @cover_state == bad`

**Review by Yazino12 — CHANGES_REQUESTED** [review]
Hello @aimalamiri ,

**To recap:**

- No linter errors ✔
- Override can_be_archived? method from parent correctly ✔
- Descriptive pull ✔

_Nice job implementing the requirements of the Task 👏🏾, But there are still some changes you need to make to proceed to the next task._

**CHECK THE COMMENTS BELOW 👇🏾👇🏾👇🏾**

File: classes/book.rb (hunk 1 above)
Comment (aimalamiri) [reply classes/book.rb]: Thanks, for referring 

I fixed the issue.

**Review by Yazino12 — APPROVED** [review]
### Hi @aimalamiri , 

- Implemented the addressed issue ✔

You did an awesome job with your tasks 🚀
Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**